### PR TITLE
fix: Add title for footer links that open in a new tab #153

### PR DESCRIPTION
**Pull request by pprev94**
I've added title in all the links with target="_blank"

### CODE REVIEW

File: src/Footer.tsx [hunk 1]
```diff
@@ -332,6 +332,7 @@ export const Footer = memo(
                                             )}
                                             target="_blank"
                                             href={`https://${domain}`}
+                                            title={`${domain} - ouvre une nouvelle fenêtre`}
```
Comment (garronej) [inline src/Footer.tsx:335]: "ouvre une nouvelle fenêtre" should be localized as well.

**Review by garronej — CHANGES_REQUESTED** [review]
Thank you

File: src/Footer.tsx (hunk 1 above)
Comment (pprev94) [reply src/Footer.tsx:335]: hi, It's fixed.

Comment (garronej) [reply src/Footer.tsx:335]:
```suggestion
                                            title={`${domain} - ${t("open new window")}`}
```

**Review by garronej — CHANGES_REQUESTED** [review]

**Comment by garronej** [comment]
Thank you!

**Pull request merged**